img-buildkite-64.json

```diff
@@ -1,4 +1,141 @@
 [
+  {
+    "name": "smartbear-alertsite",
+    "image": "img-buildkite-64/smartbear-alertsite.png",
+    "category": "Buildkite",
+    "aliases": ["alertsite"]
+  },
+  {
+    "name": "smartbear-aqtimepro",
+    "image": "img-buildkite-64/smartbear-aqtimepro.png",
+    "category": "Buildkite",
+    "aliases": ["aqtimepro"]
+  },
+  {
+    "name": "smartbear-bitbar",
+    "image": "img-buildkite-64/smartbear-bitbar.png",
+    "category": "Buildkite",
+    "aliases": ["bitbar"]
+  },
+  {
+    "name": "smartbear-bugsnag",
```

Comment:
Could you switch the name and the alias on these so the name matches the product name (like `Bugsnag` etc). Aliases can have the prefixed name instead.

It would provide better visibility for customers looking for Bugsnag emoji in readme etc as the product name themselves aren't prefixed.

Reply:
No problem - done.